src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,69 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 import { NewCommentForm } from '../NewCommentForm';
+import { getPostDetails } from '../../api/posts';
+import { getPostComments, removeComment } from '../../api/comments';
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+export const PostDetails = ({ postId }) => {
+  const [post, setPost] = useState({});
+  const [comments, setComments] = useState([]);
+  const [isHidden, hide] = useState(false);
+
+  useEffect(() => {
+    getPostDetails(postId)
+      .then(data => setPost(data));
+
+    getPostComments(postId)
+      .then(data => setComments(data));
+  }, [postId, comments]);
```

Comment:
```suggestion
  }, [postId]);
```
![infinity_loop(comments)](https://user-images.githubusercontent.com/35385361/93333545-228ae980-f82c-11ea-928c-bb31b6664b5b.gif)
забери comments з залежностей, а для того щоб оновлювались дані роби новий запит при нажиманні на кнопки додати і видалити коментар